web/src/components/organisms/StaffSelect.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Select } from 'flowbite-react';
+
+const StaffSelect = (props) => {
+    const { errors, name, register, ...rest } = props;
```

Comment:
we want options to be dynamic so lets create a parameter for it
```suggestion
    const { errors, name, register, options, ...rest } = props;
```